.github/workflows/ci.yaml

```diff
@@ -4,14 +4,14 @@ on:
   push:
     paths:
       - ".github/workflows/ci.yaml"
-      - "lib/**"
-      - "test/**"
+      - "packages/widget_toolkit/lib/**"
```

Comment:
Please use the matrix option. You can see how it's done in the bloc or rx_bloc package